test/spec/check/custom_spec.rb

```diff
@@ -27,46 +27,35 @@
     expect($media_id.nil?).to be(false)
   end
 
-  # This tests is unreliable
-  # Todo: consider fixing it or removing it
-  #
-  # CGB 2017-9-29
-  #
-  # it "should set status to media as a command", bin4: true do
-  #   media = api_create_team_project_and_claim
-  #   @driver.navigate.to media.full_url
-  #   sleep 2
+  it "should set status to media as a command", bin4: true do
+    media = api_create_team_project_and_claim
+    @driver.navigate.to media.full_url
+    sleep 2
 
-  #   # Add a status as a command
-  #   fill_field('#cmd-input', '/status In Progress')
-  #   @driver.action.send_keys(:enter).perform
-  #   sleep 5
+    # Add a status as a command
+    fill_field('#cmd-input', '/status In Progress')
+    @driver.action.send_keys(:enter).perform
+    sleep 5
 
-  #   # Verify that status was added to annotations list
-  #   expect(@driver.page_source.include?('Status')).to be(true)
+    # Verify that status was added to annotations list
+    expect(@driver.page_source.include?('Status')).to be(true)
```

Comment:
Please add a `expect(@driver.page_source.include?('Status')).to be(false)` before the `fill_field` call above - this way we know that something actually changed.

Reply:
Agree